modules/media/jsx/editForm.js

```diff
@@ -40,6 +40,7 @@ class MediaEditForm extends Component {
           dateTaken: data.mediaData.dateTaken,
           comments: data.mediaData.comments,
           hideFile: data.mediaData.hideFile,
+          language_id: data.mediaData.language_id,
```

Comment:
After reviewing all the changes, it seems like the only change needed here is simple adding this line. and if you undo all the renaming of variables, then this line becomes

```suggestion
          language: data.mediaData.language,
```

I just tested it out and it works. Is there any justification for all the other changes ?

Reply:
Yes, there was 2 defects in the module, all related to language. The fact that the label is in a separate table witrh an FK in the media table. All this was missed during reactification.

Reply:
what's the other defect ? from what I could tell the only issue was that when loading the edit form, the language was not displayed.

Reply:
`language` is for the list of possible language and `language_id` the one that the documents is tag with.

Reply:
I'm not sure I understand. After removing all changes and just changing this one line everything is functional for me.

I'll leave the decision to other reviewers.

Reply:
have you tried to edit an document meta-data without the other change? 2nd and 3rd bullet of how to test in the description.

Reply:
yes, give it a try.

Reply:
@PapillonMcGill have you had a chance to give it a try with only this line ?

Reply:
@ridz1208 You're right, minimal change required. Not sure what was the cause but with other change, the modal windows closed automatically when I click on the OK button after update during initial test. While retrying, the modal windows always stayed after clicking on the Ok button after update.